docs/en/compute/Hive.md

```diff
@@ -68,6 +68,12 @@ $ ./bin/alluxio fs copyFromLocal /path/to/ml-100k/u.user alluxio://master_hostna
 View Alluxio WebUI at `http://master_hostname:port` and you can see the directory and file Hive
 creates:
 
+> Note: Since the test data files are in text, make sure Hive is configured to read from TextFile. 
+```properties
```

Comment:
Do these need to be in XML format in hive-site.xml? Or are you recommending setting these in the Hive-CLI?